CHANGELOG.md

```diff
@@ -1,8 +1,14 @@
 ## CHANGELOG
 
-### [v2.0.0] - Version 2.0 Initial Release
+### [v2.0.3] - Minor Updates, Connection Events and Timestamps
 
-#### Summary: 
+#### Added
+
+- Add connection events for the Event Stream to allow subscription and callbacks. Attach a callback with `isy.connection_events(callback)` and receive a string with the event detail. See `constants.py` for events starting with prefix `ES_`.
+
+### [v2.0.2] - Version 2.0 Initial Release
+
+#### Summary:
```

Comment:
Thanks for cleaning up the release notes to handle the 2.0.2 release kerfuffle

Reply:
NP. I know the pain. Went through it on -beta when I originally changed everything.